content/en/docs/cluster-administration/platform-settings/notification-management/configure-email.md

```diff
@@ -1,10 +1,62 @@
 ---
-title: "Configure Email"
+title: "Configure Email Notifications"
 keywords: 'KubeSphere, Kubernetes, custom, platform'
-description: ''
-linkTitle: "Configure Email"
+description: 'Configure your email server and add recipients to receive email notifications for alerting policies.'
+linkTitle: "Configure Email Notifications"
 weight: 8721
 ---
 
-TBD
+This tutorial demonstrates how to configure your email server and add recipients, who can receive email notifications for alerting policies.
+
+## Configure the Email Server
+
+1. Log in to the web console with an account granted the role  `platform-admin`.
+
+2. Click **Platform** in the top left corner and select **Platform Settings**.
+
+3. Navigate to **Email** under **Notification Management**.
+
+   ![emai-server](/images/docs/cluster-administration/platform-settings/notification-management/configure-email/email-server.png)
+
+4. Under **Server Settings**, configure your email server by filling in the following fields.
+
+   - **SMTP Server Address**: The SMTP server address that can provide email services. The port is usually `25`.
+   - **Use SSL Secure Connection**: SSL can be used to encrypt emails, thereby improving the security of information transmitted by email. Usually you have to configure the certificate for the email server.
+   - **SMTP User**: The SMTP account.
+   - **SMTP Password**: The SMTP account password.
+   - **Sender Email Address**: The sender's email address. Customized email addresses are currently not supported.
+
+5. Click **Save**.
+
+## Add Recipients
+
+1. Under **Recipient Settings**, enter a recipient's email address and click **Add**.
+
+2. After it is added, the email address of a recipient will be listed under **Recipient Settings**. You can add multiple recipients so that all of them will be able to receive email notifications of alerts.
```

Comment:
You can add multiple recipients
=>
You can add up to 50 recipients

Reply:
Changed.